ddlite.py

```diff
@@ -680,18 +679,18 @@ def __init__(self, candidates, feats=None, gt_dict=None):
     # LF data
     self.lf_matrix = None
     self.lf_names = []
-    # Model data    
-    self.X = None
-    self._w_fit = None
-    self.w = None
+    # Model probabilities
+    self.model = None
+    self.probabilities = None
+    self.lf_marginals = None
+    self.lf_weights = None
```

Comment:
Ok I agree; maybe then stick w what you had just be consistent e.g. `lf_marginals` and `marginals`?  I dunno, I guess whatever is fine

Sure I was just mirroring your title below, that's fine, leave as `lf_weights`